screens/PostCreate.js

```diff
@@ -0,0 +1,20 @@
+import { View, StyleSheet } from "react-native";
+import { Input, Button, Card } from "react-native-elements";
+
+const PostCreate = () => {
```

Comment:
maybe this should be a CommentCreate?

ref
https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fi.stack.imgur.com%2FAyIkW.png